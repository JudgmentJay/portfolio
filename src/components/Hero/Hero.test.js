import React from 'react'
import renderer from 'react-test-renderer'

import { Hero } from '../'

it('Renders correctly', () => {
	const tree = renderer
		.create(<Hero />)
		.toJSON()

	expect(tree).toMatchSnapshot()
})
