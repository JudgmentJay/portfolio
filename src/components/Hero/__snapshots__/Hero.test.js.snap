// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<header
  className="hero"
>
  <div
    className="heroContent"
  >
    <div
      className="headers"
    >
      <h1
        className="header"
      >
        Jay Posten
      </h1>
      <h2
        className="subheader"
      >
        Software Engineer
      </h2>
    </div>
    <nav
      className="nav"
    >
      <a
        className="link"
        href="#about"
      >
        About
      </a>
      <a
        className="link"
        href="#experience"
      >
        Experience
      </a>
      <a
        className="link"
        href="#skills"
      >
        Skills
      </a>
      <a
        className="link"
        href="#projects"
      >
        Projects
      </a>
    </nav>
  </div>
</header>
`;
